sdk/nodejs/cmd/pulumi-language-nodejs/language_test.go

```diff
@@ -66,6 +69,15 @@ func (e *hostEngine) Log(_ context.Context, req *pulumirpc.LogRequest) (*pbempty
 		}
 	}
 
+	if e.previousMessage == message {
```

Comment:
We're intentionally using the shortened message here, since we don't want to keep a large amount of data in memory.